packages/lambda-powertools-middleware-log-timeout/index.js

```diff
@@ -0,0 +1,42 @@
+const Promise = require('bluebird')
```

Comment:
Do we really need to include Bluebird? I think we should be able to delay and cancel promises in native JS.

Reply:
I don't think we need promises at all - should be possible with a simple `setTimeout` and `clearTimeout`

Reply:
no, don't need `bluebird` really, it was just easy. Good shout on trimming dependencies, will replace with `setTimeout` and `clearTimeout` instead